examples/stream-last-value-table/src/components/Row/index.tsx

```diff
@@ -11,23 +11,31 @@ interface IRowProps {
   idx?: number;
 }
 
+const WINDOW_TELEOP_MIN_WIDTH = 301;
+const TIMEOUT_DEBOUNCER = 100;
+
 export const Row: FC<IRowProps> = (props: any) => {
   const context = useFormant();
   const config = context.configuration as IConfiguration;
+  const timeoutId = useRef<any>(null);
   const { leftValue, rightValue, state, height, fullWidth } = props;
-  const [teleopMode, setTeleopMode] = useState(false);
+  const [teleopMode, setTeleopMode] = useState(() => window.innerWidth < WINDOW_TELEOP_MIN_WIDTH);
```

Comment:
Setting initial state based on external variables (window width), that's why I prefer a computing function.